addons/medical_gui/gui.hpp

```diff
@@ -216,7 +216,7 @@ class ACE_Medical_Menu {
         };
         class Triage: RscActivePicture {
             idc = IDC_TRIAGE;
-            onButtonClick = QUOTE(GVAR(selectedCategory) = 'triage');
+            onButtonClick = QUOTE(if (GVAR(selectedCategory) isEqualTo 'triage') then {GVAR(selectedCategory) = 'triage'; GVAR(page) = GVAR(page) + 1;} else {GVAR(selectedCategory) = 'triage'};);
```

Comment:
Should be moved to a function IMO, `onCategorySelected` or smth.

Reply:
Resolved in latest commits